substrate/frame/revive/src/storage.rs

```diff
@@ -566,6 +567,24 @@ impl<T: Config> DeletionQueueManager<T> {
 	}
 }
 
+/// Debugging settings that can be configured when DebugEnabled config is true.
+#[derive(
+	Default,
+	Encode,
+	Decode,
+	Clone,
+	PartialEq,
+	RuntimeDebug,
+	TypeInfo,
+	MaxEncodedLen,
+	Serialize,
+	Deserialize,
+)]
+pub struct DebugSettings {
+	/// Whether to allow unlimited contract size.
+	pub allow_unlimited_contract_size: bool,
+}
```

Comment:
Should be in its own top level debug module. Does not make sense in the storage module.